pkg/actions/diagnostics.go

```diff
@@ -409,10 +407,9 @@ func findIntellijDirectory(inDir string) string {
 }
 
 func gatherCodewindIntellijLogs(codewindIntellijLogDir string) {
-	logDG("Collecting Intellij logs ... ")
 	intellijLogsDir := codewindIntellijLogDir
 	if intellijLogsDir == "" {
-		// attempt to use default path
+		logDG("Attempting to collect Intellij logs from default location ... ")
```

Comment:
Again if this was after the switch then we could tell the user where we were looking. You could also put it inside the `if len(` block so it was logged even if they specified a directory.

Reply:
OK, think I've got those in the right place now!